contracts/axelar-gateway/src/storage_types.rs

```diff
@@ -19,11 +19,8 @@ pub enum MessageApprovalValue {
 #[derive(Clone, Debug)]
 pub enum DataKey {
     /// Gateway
-    Owner,
```

Comment:
as part of doing code reuse, we wanted to understand the impact of using different DataKey types for the same contract. If XDR encoding is used, then unique enum variant names should be enough to avoid collisions. But if something else is done internally with the key, then we'd need to investigate